build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -225,5 +225,12 @@
           DestinationFiles="@(FilesToCopy->'$(StageSymbolsDirectory)\%(RecursiveDir)%(Filename)%(Extension)')" />
 
       <Delete Files="@(PdbsToClean)" />
+
+      <!-- workaround for https://github.com/Microsoft/msbuild/issues/872 -->
+      <Copy SourceFiles="$(RepoRoot)/resources/MSBuild.exe.config"
+            DestinationFiles="$(SdkOutputDirectory)/MSBuild.exe.config" />
```

Comment:
This would be a nice idea and then the migration tool would just move it to a copy to output item in the csproject, right?